slacker/commands/emoji/emoji_list_command.py

```diff
@@ -2,9 +2,8 @@
 import json
 import requests
 
-from argparse import ArgumentParser
-
 from slacker.commands.command import Command
+from slacker.commands.argument_parser import ArgumentParser
```

Comment:
I included the use of our own `slacker.commands.argument_parser.ArgumentParser`.